ifrs17-template/Files/DataNodes/DataNodes_CH.csv

```diff
@@ -3,7 +3,7 @@ ReportingNode,,,,,,
 CH,,,,,,
 ,,,,,,
 @@InsurancePortfolio,,,,,,
-SystemName,DisplayName,ContractualCurrency,LineOfBusiness,ValuationApproach,OciType,
+SystemName,DisplayName,ContractualCurrency,LineOfBusiness,ValuationApproach,OciType,Scenario,
```

Comment:
Remove Scenario from the columns of Data Node csv files because it shouldn't be mandatory. Apply to all recurrences.

Reply:
done